docs/features/plugin/start_coding.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 5
+title: "🔧 Coding Functions"
```

Comment:
`Shared Components`, Better suggestions welcome.

Reply:
I was thinking about some name that will show that it is related to making a new Function. I will change it to this for now.